### PR TITLE
GRF-874: add init template choice

### PR DESCRIPTION
**Pull request by pchasle**
- Add template initialization choice
- Replaced custom CSS to fully use the DSM `Placeholder` component

Choice to start from attribute set or create 1st attribute:
localhost_3000_

Confirmation modal:
localhost_3000_ (1)

No attribute on category page:
localhost_3000_ (2)

### CODE REVIEW

**Review by jeremyBeaucousinAkeneo — APPROVED** [review]

File: src/Akeneo/Category/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml [hunk 1]
```diff
@@ -102,6 +102,27 @@ akeneo.category:
             saving: Saving
             editing: Editing
             errors: Errors not saved
+        initialize:
+            title: Add category attributes
+            message: Define content of your categories by adding Text, Text Area or Image attributes.
+            choice_message: Start by creating a first attribute according to your needs, or load a set 13 suggested attributes.
```
Comment (etiennekerbastard) [inline src/Akeneo/Category/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml:108]: It maches the mock-up however I think there is a typo in the mock-up. It should be "...load a set **_of_** 13 suggested attributes"

File: src/Akeneo/Category/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml [hunk 2]
```diff
@@ -102,6 +102,27 @@ akeneo.category:
...
+            button:
+                load: Load a set of attributes
+                create: Create 1st attribute
+        load_attribute_set:
+            section_title: Load set of attributes
+            title: Load a set of suggested attributes
+            description: Easily start the configuration of enriched categories by loading a set of suggested attributes.
+            content_description: This set contains 13 attributes :
```
Comment (etiennekerbastard) [inline src/Akeneo/Category/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml:116]: Same here, it's "chipotage" but in English there should be no space between "attributes" and ":"

File: src/Akeneo/Category/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml [hunk 3]
```diff
@@ -102,6 +102,27 @@ akeneo.category:
...
+            content:
+                description_attributes: 2 Text Area attributes for Long and Short category descriptions
+                url_attributes: 1 Text attribute for URL information
+                image_attributes: 3 Images and 3 Text attributes for image and their alt-text information
+                seo_attributes: 2 Text and 2 Text Area for SEO information
+            confirmation_message: Are you sure want to load this set of attributes ?
```
Comment (etiennekerbastard) [inline src/Akeneo/Category/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml:122]: should be "...attributes?" with no space as above

File: src/Akeneo/Category/front/src/feature/components/templates/EditTemplateAttributesForm/InitializeTemplateChoice.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+interface Props {
+  templateId: string;
+}
+
+export const InitializeTemplateChoice = ({templateId}: Props) => {
+  const [isAddTemplateAttributeModalOpen, openAddTemplateAttributeModal, closeAddTemplateAttributeModal] =
+    useBooleanState(false);
+
+  const [isLoadAttributeSetModalIpen, openLoadAttributeSetModalIpen, closeLoadAttributeSetModalIpen] =
```
Comment (etiennekerbastard) [inline src/Akeneo/Category/front/src/feature/components/templates/EditTemplateAttributesForm/InitializeTemplateChoice.tsx:15]:
```suggestion
  const [isLoadAttributeSetModalOpen, openLoadAttributeSetModal, closeLoadAttributeSetModal] =
```

**Review by etiennekerbastard — APPROVED** [review]

File: src/Akeneo/Category/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml (hunk 1 above)
Comment (pchasle) [reply src/Akeneo/Category/back/Infrastructure/Symfony/Resources/translations/jsmessages.en_US.yml:108]: Thanks for the proof-reading :pray:

**Pull request merged**